pyblish_ftrack/plugins/conform_ftrack_component_preview.py

```diff
@@ -0,0 +1,43 @@
+import pyblish.api
+import os
+import ftrack
+
+@pyblish.api.log
+class FtrackUploadPreview(pyblish.api.Conformer):
+    """Creates Preview component within supplied version, also makes this preview reviewable.
+
+    Expected data members:
+    'publishedFile' - path that will be saved as a component
+    'ftrackVersionID' - ID of a version where component should be created
```

Comment:
The standard I had in mind is to hijack the `Arguments:` portion of Google Napoleon docstrings, as they aren't used for anything.

```python
"""
    Arguments:
        publishedFile (string): path that will be saved as a component
        ftrackVersionID (string, optional): ID of a version where component should be created
"""
```